otter/cloud_client.py

```diff
@@ -507,6 +524,25 @@ class NovaComputeFaultError(Exception):
     """
 
 
+@attributes([])
```

Comment:
@manishtomar How do you feel about `ExceptionWithMessage`?  So long as `@attr` will used, all of them have equality, the one with message is just so, like all the other builtin exceptions, our exceptions will also take a message in the `__init__`.